src/Collections/TerminusCollection.php

```diff
@@ -262,7 +277,7 @@ public function getMemberList($key = 'id', $value = 'name')
      */
     protected function getMembers()
     {
-        if (empty($this->models)) {
+        if ($this->models === null) {
```

Comment:
It begins.
*pops an umbrella and prays that none of the equals signs lands on it with its points down*